src/js/utils/prop-types.js

```diff
@@ -1,7 +1,8 @@
 import { PropTypes } from 'react-desc';
 
 export const a11yTitlePropType = PropTypes.string.description(
-  'Custom title to be used by screen readers.',
+  `Custom label to be used by screen readers. When provided, an aria-label will
+   be added to the element.`,
```

Comment:
@ericsoderberghp or @alansouzati, was there a reason we avoided mentioning that `a11yTitle` is actually being translated to `aria-label` under the hood?

Reply:
I can't remember. :(